server/portal/libs/agave/models/systems/base.py

```diff
@@ -354,28 +354,31 @@ def update_role(self, username, role):
         )
 
     def test(self):
-        """Test system
+        """Test storage system system
 
-        .. todo::
-            As of 05/2018 this only tests storage systems
-            by doing a `files-listing` on it.
-            What is a good way to test exec systems?
+        Test is a file listing of the system. If a system is missing, TAPIS *should* return
+        a 404.  Note though, that TAPIS sometimes returns an error that isn't a 404.
+
+        Execution systems are not supported.
         """
-        result = 'FAIL'
         success = True
-        # if self.type == self.TYPES.STORAGE:
+        result = 'SUCCESS'
```

Comment:
Is `result` ever used?

Reply:
It is not used in the system creation step but it is used in other places it is used (`server/portal/apps/accounts/managers/accounts.py`).